src/source/GaussFunction.cpp

```diff
@@ -36,18 +35,12 @@ GaussFunction::~GaussFunction() {
 }
 
 void GaussFunction::update_source(Field *out, real t_cur) {
-    size_t size = Domain::getInstance()->get_size();
-    auto data_out = out->data;
-    auto data_spatial = m_field_spatial_values->data;
     auto time_val = get_time_value(t_cur);
+    out->copy_data(*m_field_spatial_values);
```

Comment:
same line in Cube.cpp:12 but with *:
`(*out).copy_data(*m_source_field);`
wondering where the difference is?

Reply:
the notation in Cube was wrong. fixed in 8cba1aa